kerykeion/fixed_stars/discovery_factory.py

```
@@ -2,59 +2,134 @@
     try:
         with open(catalog_path, "r", encoding="utf-8", errors="replace") as f:
             for line in f:
                 line = line.strip()
                 if not line or line.startswith("#"):
                     continue
                 # sefstars.txt format: "name, nomenclature, equinox, RA, Dec, ..."
-                # The first field (before the first comma) is the star name
                 name = line.split(",")[0].strip()
                 if name:
                     names.append(name)
     except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)